### PR TITLE
feat(deps): Add Nextcloud 28 support on master

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<name>Two-Factor TOTP Provider</name>
 	<summary>TOTP two-factor provider</summary>
 	<description>A Two-Factor-Auth Provider for TOTP (RFC 6238)</description>
-	<version>9.0.0-beta.2</version>
+	<version>9.1.0-dev</version>
```
Comment (ChristophWurst) [inline appinfo/info.xml:8]:
```suggestion
	<version>10.0.0-dev</version>
```

https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/release_process.html#major-update

**Review by ChristophWurst — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "twofactor_totp",
   "description": "Nextcloud TwoFactor TOTP",
-  "version": "9.0.0-beta2",
+  "version": "9.1.0-dev",
```
[the remark this answers is not in the document]
Comment (st3iny) [reply package.json]: Done

File: appinfo/info.xml (hunk 1 above)
Comment (st3iny) [reply appinfo/info.xml]: Done

**Comment by st3iny** [comment]
Test failures are unrelated.

**Pull request merged**